shared/wallets/wallet/settings/index.js

```diff
@@ -56,92 +56,98 @@ const HoverText = Styles.isMobile
 
 const AccountSettings = (props: SettingsProps) => {
   return (
-    <Kb.Box2 direction="vertical" fullWidth={true}>
-      <Kb.HeaderHocHeader title="Settings" onBack={props.onBack} headerStyle={styles.header} />
-      <Kb.Box2 direction="vertical" style={styles.settingsPage} fullWidth={true}>
-        <Kb.Text type="BodySmallSemibold">Account name</Kb.Text>
-        <Kb.ClickableBox onClick={props.onEditName} style={styles.nameBox}>
-          <HoverText type="BodySemibold">{props.name}</HoverText>
-          <Kb.Icon style={Kb.iconCastPlatformStyles(styles.icon)} type="iconfont-edit" />
-        </Kb.ClickableBox>
-        <Kb.Box2 direction="vertical" fullWidth={true} gap="tiny">
-          <Kb.Text type="BodySmallSemibold">Stellar address</Kb.Text>
-          <Kb.CopyText text={props.accountID} containerStyle={styles.accountIDContainer} />
-        </Kb.Box2>
-        <Kb.Box2 direction="vertical" style={styles.sectionLabel}>
-          <Kb.Text type="BodySmallSemibold">Identity</Kb.Text>
-        </Kb.Box2>
-        <Kb.Box2 direction="horizontal" fullWidth={true} gap="tiny" style={styles.accountBox}>
-          {props.isDefault ? (
-            <Kb.Avatar size={32} username={props.user} />
-          ) : (
-            <Kb.Icon type="icon-placeholder-secret-user-32" style={{height: 32, width: 32}} />
-          )}
-          <Kb.Box2 direction="vertical">
-            <Kb.Text type="Body">
-              {props.isDefault ? 'This is your default Keybase account.' : 'This is a secondary account.'}
-            </Kb.Text>
-            <Kb.Text type="BodySmall">
-              {props.isDefault
-                ? 'All transactions and overall activity are tied to your Keybase identity.'
-                : 'Transactions will be tied to your Stellar public address only.'}
-            </Kb.Text>
-            {!props.isDefault && (
-              <Kb.Text type="BodySmallPrimaryLink" onClick={props.onSetDefault}>
-                Set as default Keybase account
-              </Kb.Text>
+    <Kb.ScrollView style={styles.scrollView}>
+      <Kb.Box2 direction="vertical" fullWidth={true}>
+        <Kb.HeaderHocHeader title="Settings" onBack={props.onBack} headerStyle={styles.header} />
```

Comment:
This header scrolls with the page, maybe we should use real `HeaderHoc` or put the scrollview below this?